app/controllers/application_controller.rb

```diff
@@ -1,8 +1,11 @@
 class ApplicationController < ActionController::Base
+  include Pundit
   protect_from_forgery with: :exception
   helper_method :current_person
 
   def current_person
     @current_person ||= Person.find(session[:person_id]) if session[:person_id]
   end
+
+  alias pundit_user current_person
```

Comment:
We have to I believe. The gem searches for a `current_user`. If you are using a different naming  convention (`current_person`) the docs say you should just alias it this way.